### ISSUE TITLE
returns more than one topic in the sentence

### ISSUE THREAD

**Issue by naokityokoyama** [MaartenGr/BERTopic#1279]
Is it possible for bert to return more than one topic in the same sentence?

because after training the model it returns a vector with only 1 value

#vectorizer
ctfidf  =  ClassTfidfTransformer()
bow  =  CountVectorizer(ngram_range=(1,3))

#Reduce dimensionality
umap_model = UMAP(n_neighbors=15, n_components=5, min_dist=0.0, metric='cosine', random_state=42)

#Cluster reduced embeddings
hdbscan_model = HDBSCAN(min_cluster_size=15, metric='euclidean', cluster_selection_method='eom', prediction_data=True)

#model
#embedding_model = 'xlm-r-bert-base-nli-stsb-mean-tokens'
embedding_model = 'all-MiniLM-L6-v2'

model = BERTopic(language='Portuguese', 
                 embedding_model = embedding_model, 
                 umap_model = umap_model,
                 hdbscan_model = hdbscan_model,
                 vectorizer_model = bow,
                 ctfidf_model=ctfidf,
                calculate_probabilities=True,
                 verbose=True)

topics, prob = model.fit_transform(corpus)

topics[0]
1
topic[1]
0

**Comment by MaartenGr** [comment]
Yes, you can use [`.approximate_distribution `](https://maartengr.github.io/BERTopic/getting_started/distribution/distribution.html) for that.

**Comment by naokityokoyama** [comment]
thanks 
it worked !!!!!
